src/System.Globalization/tests/CultureInfo/CultureInfoCalendar.cs

```diff
@@ -8,13 +8,20 @@ namespace System.Globalization.Tests
 {
     public class CultureInfoCalendar
     {
-        [Theory]
-        [InlineData("en-US", typeof(GregorianCalendar))]
-        [InlineData("th-TH", typeof(ThaiBuddhistCalendar))]
-        public void Calendar(string locale, Type expected)
+        [Fact]
+        public void Calendar_Get_InvariantInfo()
         {
-            CultureInfo cultureInfo = new CultureInfo(locale);
-            Assert.IsType(expected, cultureInfo.Calendar);
```

Comment:
@stephentoub it is wrong test, if the user configured the culture with other default calendar that will break this test. testing with invariant is more reliable. also we may add test to check if the calendar is one of the optional calendars for the culture instead of testing it is the default calendar